base/sysinfo.jl

```diff
@@ -162,7 +162,7 @@ total_memory() = ccall(:uv_get_total_memory, UInt64, ())
 Get the process title. On some systems, will always return empty string. (not exported)
 """
 function get_process_title()
-    buf = zeros(UInt8, 512)
+    buf = Vector{UInt8}(512)
```

Comment:
`StringVector` is just a special constructor for `Vector{UInt8}` so you can resize it like any other `Vector{UInt8}`.